docs/contributing/guidelines/license.md

```diff
@@ -103,6 +103,22 @@ If you decide to use a different license for your work, follow the same pattern:
 
 - If more than one license applies to the source file, use an SPDX license expression (see [SPDX Specification, Appendix IV](https://spdx.org/spdx-specification-21-web-version#h.jxpfx0ykyb60)), to reflect the presence of multiple licenses in your *LICENSE.md* file and in each source file.
 
+## License check in pull requests
+
+We use scancode-toolkit to enforce the license in binaries and libraries. Please see "Continuous integration (CI) testing" section for more information.
```

Comment:
I can move `Please see "Continuous integration (CI) testing" section for more information.` to the end of this section, shall I?

Reply:
I'm agnostic on this one.